testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/model/ReadFullModelTestCase.java

```diff
@@ -26,63 +26,55 @@
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RECURSIVE;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
 
-import org.jboss.arquillian.container.test.api.ContainerController;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.test.api.ArquillianResource;
+import javax.inject.Inject;
+
 import org.jboss.as.controller.PathAddress;
-import org.jboss.as.controller.client.ModelControllerClient;
 import org.jboss.as.controller.operations.common.Util;
-import org.jboss.as.model.test.ModelTestUtils;
-import org.jboss.as.test.shared.ServerReload;
-import org.jboss.as.test.shared.TestSuiteEnvironment;
 import org.jboss.dmr.ModelNode;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerControl;
+import org.wildfly.core.testrunner.ServerController;
+import org.wildfly.core.testrunner.WildflyTestRunner;
 
 /**
  * Ensures that the full model including runtime resources/attributes can be read in both normal and admin-only mode
  * for the fullest server config possible
  *
  * @author Kabir Khan
+ * @author Tomaz Cerar
  */
-@RunWith(Arquillian.class)
-@RunAsClient
+@ServerControl(manual = true)
+@RunWith(WildflyTestRunner.class)
 public class ReadFullModelTestCase {
 
     //This is the full-ha setup which is the fullest config we have
     //Reuse this rather than a setup
-    public static final String SERVER = "jbossas-messaging-ha-server1";
```

Comment:
this was actually wrong before as it only started server with this name (in our target/<name-of-server>) folder not with standalone-full-ha.xml configuration.